src/Maestro/Maestro.Data/BuildAssetRegistryContext.cs

```diff
@@ -72,12 +73,43 @@ public class BuildAssetRegistryContext : IdentityDbContext<ApplicationUser, Iden
         public DbSet<Repository> Repositories { get; set; }
         public DbSet<RepositoryBranch> RepositoryBranches { get; set; }
         public DbSet<RepositoryBranchUpdate> RepositoryBranchUpdates { get; set; }
-        public DbQuery<RepositoryBranchUpdateHistoryEntry> RepositoryBranchUpdateHistory { get; set; }
-        public DbQuery<SubscriptionUpdateHistoryEntry> SubscriptionUpdateHistory { get; set; }
         public DbSet<DependencyFlowEvent> DependencyFlowEvents { get; set; }
         public DbSet<GoalTime> GoalTime { get; set; }
         public DbSet<LongestBuildPath> LongestBuildPaths { get; set; }
 
+        public virtual IQueryable<RepositoryBranchUpdateHistoryEntry> RepositoryBranchUpdateHistory => RepositoryBranchUpdates.FromSqlRaw(@"
+SELECT * FROM [RepositoryBranchUpdates]
+FOR SYSTEM_TIME ALL
+")
+            .Select(
+                u => new RepositoryBranchUpdateHistoryEntry
+                {
+                    Repository = u.RepositoryName,
+                    Branch = u.BranchName,
+                    Action = u.Action,
+                    Success = u.Success,
+                    ErrorMessage = u.ErrorMessage,
+                    Method = u.Method,
+                    Arguments = u.Arguments,
+                    Timestamp = EF.Property<DateTime>(u, "SysStartTime")
+                });
+
+        public IQueryable<SubscriptionUpdateHistoryEntry> SubscriptionUpdateHistory => SubscriptionUpdates.FromSqlRaw(@"
+SELECT * FROM [SubscriptionUpdates]
+FOR SYSTEM_TIME ALL
+")
+            .Select(
+                u => new SubscriptionUpdateHistoryEntry
+                {
+                    SubscriptionId = u.SubscriptionId,
+                    Action = u.Action,
+                    Success = u.Success,
+                    ErrorMessage = u.ErrorMessage,
+                    Method = u.Method,
+                    Arguments = u.Arguments,
+                    Timestamp = EF.Property<DateTime>(u, "SysStartTime")
+                });
+
```

Comment:
How are these changes related to the version upgrade to 3.1?

Reply:
The previous thing, DbQuery, isn't around anymore.  All of these changes are because of breaking changes because of standardizing from 2.1 to 3.1.  There are no functional changes anywhere that weren't necessitated by the change.